flexible_event_config.md

```diff
@@ -116,6 +223,7 @@ Every trigger registration will match with at most one trigger spec and update i
 * Apply global attribution filters
 * For every trigger spec:
   * Evaluate the `event_trigger_data` on the spec to find a match, using the spec’s `event_reporting_window`
+    * The global `event_reporting_windows` (from Phase 1) will act as a default value in case any trigger spec is the missing `event_report_windows` sub-field
```

Comment:
```suggestion
    * The top level `event_reporting_windows` will act as a default value in case any trigger spec is the missing `event_report_windows` sub-field
```